### ISSUE TITLE
Uncaught ReferenceError: sortable is not defined

### ISSUE THREAD

**Issue by larsgust** [freescout-helpdesk/freescout#1631]
The function sortable is not defined in the javascript file js/builds/xxxxxx...yyy.js

The error appears when entering the page that lists articles in the Knowledge Base admin page and you select a category other than All.

I can see that the function is not defined in the javascript file. It is however defined on your demo site so it might be a combination of the version of Freescout and some additional (official) modules.

Freescout version 1.7.27 

The following modules are also activated:

API & Webshooks version Version: 1.0.30
End-User-Portal version 1.0.20
Knowledge Base version 1.0.27
Workflows 1.0.21

**Comment by freescout-helpdesk** [comment]
Most likely permissions issue: https://github.com/freescout-helpdesk/freescout/wiki/Installation-Guide#-permissions-issue

**Comment by larsgust** [comment]
I don't think it is a permission issue. Did a complete re-installation today just to be sure that all permissions are correct. This time I used the web installer. The installation succeeds without any warnings.

The permission check under /system/status says that everything is correct. Everything works except the knowledge base. It gives javascript error "sortable is not defined" when editing articles (/mailbox/1/knowledge-base/articles).

I can see that on your demo site sortable is defined in the build javascript file as:
  "var sortable=var sortable=(function(){'use stri..."
  
grep "sortable=" . -R in the folder where I have installed freescout-dist gives no match.

Any help to solve this will be highly appreciated.

**Comment by larsgust** [comment]
Which file is expected to include the definition of the sortable method?

**Comment by larsgust** [comment]
As a temporary workaround waiting for a better solution I have managed to get it to work by identifying the javascript on your demo site for the sortable plugin that is missing in my installation, it starts with `var sortable=` and ends with `return sortable;}());`.

I then copied that javascript code and appended it to the file `public/js/jquery.js` in my installation (to avoid having to modify the blade templates or anything else for the plugin, I wanted to change as little as possible). Then I deactivated the Knowledge base plugin in my installation, deleted it and finally reinstalled and activated it again. Then it works, the javascript that I added is now included in the javascript in the `public/js/build` folder.

Of course this is not the real fix, only a temporary solution until this is resolved in a better way. I did also try to uninstall and reinstalled the Knowledge base plugin before modifying any javascript but that did not help. So it is quite clear that the javascript sortable plugin is missing in my installation.

Any ideas?
